libraries/core-react/src/Menu/MenuItem.tsx

```diff
@@ -121,7 +121,10 @@ export const MenuItem = React.memo(
     return (
       <ListItem
         {...props}
-        ref={useCombinedRefs(ref, (el: HTMLElement) => isFocused && el.focus())}
+        ref={useCombinedRefs<HTMLLIElement>(
+          ref,
+          (el: HTMLElement) => isFocused && el.focus(),
```

Comment:
This looks to be the wrong type (from ts convert). `el` argument should be `HTMLLIElement`? Might aswell fix it now since you are here 😅